configs/openSUSE/security.toml

```diff
@@ -26,15 +26,29 @@ Locations = [
     "/usr/share/polkit-1/rules.d/"
 ]
 
-[FileDigestGroup.atop."boo#12345"]
+[FileDigestGroup.file-signature."boo#12345"]
```

Comment:
We now lost the coupling between whitelisting and package again.

Reply:
I've used a better TOML syntax, is it still a problem?

Reply:
Hmm actually I can't see the difference to before. Where is the package coupling now?